### ISSUE TITLE
[$10][MSFT-37] The NVDA doesn't read "Data Science" and "Filters" buttons when user try to mouse hover or click on them

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2759]
**Steps to Reproduce**
1. Navigate to https://www.topcoder.com/challenges?
2. Navigate to "Design/Development/Data Science/Filters" button and try to mouse hover or click on them

**Expected Result**
The NVDA should read "Data Science" and "Filters" buttons when user try to mouse hover or click on them

**Actual Result**
The NVDA doesn't read "Data Science" and "Filters" buttons when user try to mouse hover or click on them

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.2.3 Audio Description or Media Alternative (Prerecorded)
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F4

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096071 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096071 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096071 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by sushilshinde** [comment]
@r0hit-gupta @anonymousjaggu why review label is removed I see code review passed here 
https://github.com/topcoder-platform/community-app/pull/2962

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097522 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097522 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by tosha5252** [comment]
Readout is still inconsistent when hovering with mouse.

Env: Windows 10
Browser: Chrome

**Comment by lakshmiathreya** [comment]
@tosha5252 pls check once more - it is working for me Windows 10/Chrome/NVDA - reading out Data Science and Filters consistently...

**Comment by lakshmiathreya** [comment]
Moving to Pass based on verification with NVDA ... fyi - @tosha5252 @drasticdpk

**Comment by lakshmiathreya** [comment]
Verified on MacOS Chrome VoiceOver and Win10 Chrome NVDA - reading out Data Science and Filters as expected. Closing issue.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097522<br/><br/>```This is an automated message for crazyk via Topcoder X```
